docs/users/providers/DDEV-Live.md

```diff
@@ -3,3 +3,26 @@
 DDEV provides a hosting solution, [DDEV-Live](http://dash.ddev.com/), as part of the full DDEV development and deployment platform. You can use DDEV-Live with DDEV-Local as you would use any hosting provider, by pushing code to Github, exporting databases to a file and then pushing them up, etc.
 
 Visit the [documentation pages for DDEV-Live](https://docs.ddev.com/getting-started/) to get started with Drupal, TYPO3, or other PHP projects.
+
+### DDEV-Live Quickstart
+
+If you have ddev installed, and have a DDEV-Live account with an active WordPress, Drupal 7, or Drupal 8 site, you can follow this quick start guide to spin up a DDEV-Live project locally. If you don't already have a DDEV-Live account, a free short-term trial is available on signup.
+
+1. Authenticate with DDEV-Live.
+
+    a. Login to your DDEV-Live Dashboard, and [obtain the DDEV API token](https://dash.ddev.com/settings/integration) for ddev to use.
+
+    b. Run `ddev auth ddev-live <YOUR TOKEN>` (This is a one-time operation, and configures ddev to work with all the sites on your account.) (You can also set the default DDEV-Live "org" with `ddev auth ddev-live --default-org=<yourorg> <token>`)
```

Comment:
```suggestion
    b. Run `ddev auth ddev-live <YOUR TOKEN>` (This is a one-time operation, which configures ddev to work with all the sites on your DDEV-Live account.) (You can also set the default DDEV-Live [organization](https://docs.ddev.com/organizations/) for ddev to work with using `ddev auth ddev-live --default-org=<yourorg> <token>`)
```

Reply:
This threw me off a little because I'm used to the syntax for setting the default org in ddev-live `ddev-live auth --default-org ddev-demo`
Any chance those syntaxes could be reconciled at some point?

Reply:
Well, `ddev auth <provider>` has existed a long time, I can't help it that ddev-live didn't follow it. It is default-org on both ddev and ddev-live auth though.